[Docs]: Section "Getting Code" in Contributing docs is missing some steps

### Page(s)

https://github.com/microsoft/playwright/blob/main/CONTRIBUTING.md#getting-code

### Description

I wanted to try to run the Playwright repo locally to test some settings. I was following the Contributing docs, but despite doing precisely as described in "Getting Code" section I cannot run the project.
I'm using:
```
node: v16.15.1
npm: 9.3.0
```
According to docs that should be alright.
Issue is with building. I get an error:
```
playwright % npm run build

> playwright-internal@1.42.0-next build
> node utils/build/build.js

==== Running npm ci --save=false --fund=false --audit=false in /Users/jonah/Developer/original_playwright/playwright/tests/playwright-test/stable-test-runner
npm ERR! code ERR_FS_EISDIR
npm ERR! syscall rm
npm ERR! path /Users/jonah/Developer/original_playwright/playwright/tests/playwright-test/stable-test-runner/node_modules/.bin
npm ERR! errno 21
npm ERR! Path is a directory: rm returned EISDIR (is a directory) /Users/jonah/Developer/original_playwright/playwright/tests/playwright-test/stable-test-runner/node_modules/.bin
```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -19,7 +19,7 @@ We strongly recommend that you open an issue before beginning any code modificat
 
 ### Getting Code
 
-Make sure you're running Node.js 16+ and NPM 8+, to verify and upgrade NPM do:
+Make sure you're running Node.js 18+ and NPM 8+, to verify and upgrade NPM do:
 
 ```bash
 node --version

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -19,7 +19,7 @@ We strongly recommend that you open an issue before beginning any code modificat
 
 ### Getting Code
 
-Make sure you're running Node.js 18+ and NPM 8+, to verify and upgrade NPM do:
+Make sure you're running Node.js 20 to verify and upgrade NPM do:
 
 ```bash
 node --version

